doc/nrf/app_dev/bootloaders_dfu/index.rst

```diff
@@ -8,40 +7,34 @@ Bootloaders and DFU
-+--------------------------+-------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------------+-------------------------------------------------------------------------------------------+
-| Slot characteristics     | Equal primary and secondary slot sizes lead to high memory overhead.    | Single DFU partition with customizable component sizes.                                                           | Highly efficient due to immutable design.                                                 |
-+--------------------------+-------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------------+-------------------------------------------------------------------------------------------+
-| Slot definition          | Static definition; challenging to change post-deployment.               | Technical possibility to change component definitions between updates.                                            | Fixed and immutable; no post-deployment changes.                                          |
-+--------------------------+-------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------------+-------------------------------------------------------------------------------------------+
-| Invocation process       | Limited customization through metadata.                                 | Flexible customization within the manifest (metadata).                                                            | Authenticated firmware execution; no update mechanism post-boot.                          |
-+--------------------------+-------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------------+-------------------------------------------------------------------------------------------+
-| Flash memory layout      | Specific allocations for primary and secondary slots.                   | Single DFU partition with flexible component slots.                                                               | OTP regions for provisioned data; specific layout for boot and application partitions.    |
-+--------------------------+-------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------------+-------------------------------------------------------------------------------------------+
+Depending on the device, you need to use different bootloader and DFU solutions, use :ref:`MCUboot and nRF Secure Immutable Bootloader (NSIB) <ug_bootloader_mcuboot_nsib>`.
```

Comment:
This sentence sounds awkward to me.
Maybe we can remove the `use ...` part and use the link to the user guide inside `MCUboot and |NSIB| architecture comparison` ?

Reply:
I will remove the "use".